cpp/src/io/utilities/column_buffer.cpp

```diff
@@ -265,6 +296,34 @@ std::unique_ptr<column> empty_like(column_buffer& buffer,
   }
 }
 
-}  // namespace detail
-}  // namespace io
-}  // namespace cudf
+using pointer_type = gather_column_buffer;
+using string_type  = inline_column_buffer;
+
+using pointer_column_buffer = column_buffer_base<pointer_type>;
+using string_column_buffer  = column_buffer_base<string_type>;
+
+template std::unique_ptr<column> make_column<string_type>(
+  string_column_buffer& buffer,
+  column_name_info* schema_info,
+  std::optional<reader_column_schema> const& schema,
+  rmm::cuda_stream_view stream);
+
+template std::unique_ptr<column> make_column<pointer_type>(
+  pointer_column_buffer& buffer,
+  column_name_info* schema_info,
+  std::optional<reader_column_schema> const& schema,
+  rmm::cuda_stream_view stream);
+
+template std::unique_ptr<column> empty_like<string_type>(string_column_buffer& buffer,
+                                                         column_name_info* schema_info,
+                                                         rmm::cuda_stream_view stream,
+                                                         rmm::mr::device_memory_resource* mr);
+
+template std::unique_ptr<column> empty_like<pointer_type>(pointer_column_buffer& buffer,
+                                                          column_name_info* schema_info,
+                                                          rmm::cuda_stream_view stream,
+                                                          rmm::mr::device_memory_resource* mr);
+
+template class column_buffer_base<pointer_type>;
+template class column_buffer_base<string_type>;
```

Comment:
```suggestion
```

should be a leftover

Reply:
The forced template instantiations are actually needed.  The `using` defs are just sugar.  Would you prefer fully qualified names for the rest?

Reply:
Ah, you are right. Using aliases is totally fine. Please ignore my previous comment.